ingestion/src/metadata/ingestion/source/dashboard/tableau/client.py

```diff
@@ -122,11 +130,79 @@ def get_charts(self) -> List[TableauChart]:
             )
         ]
 
+    def get_workbook_charts(self, dashboard_id: str) -> Optional[List[TableauChart]]:
+        """
+        Get the charts for a workbook
+        """
+        try:
+            return [
+                TableauChart(**chart)
+                for chart in self._client.query_views_for_workbook(
+                    workbook_id=dashboard_id,
+                    parameter_dict=TABLEAU_GET_VIEWS_PARAM_DICT,
+                ).json()["views"]["view"]
+            ]
+        except Exception as exc:
+            logger.debug(traceback.format_exc())
+            logger.warning(
+                f"Error processing charts for dashboard [{dashboard_id}]: {exc}"
+            )
+        return None
+
+    def test_api_version(self):
```

Comment:
sg, thanks. Let's add this as a comment in the JSON Schema/connection code to make sure we don't forget 🙏

Reply:
Added it as a comment in the test connection [steps](https://github.com/open-metadata/OpenMetadata/pull/14548/files#diff-ffa265b0541632f566c9cdee95eba775215013a62e055c39ebe02c1faaf6a59cR81). Since it'll be easier to reference it from there